<launch>

  <node name="ardu_remoter_pub" pkg="bit_control_tool" type="ardu_remoter_pub" output="screen" respawn="false">
  <param name="port"       value="/dev/remo_dev"/>  
  <param name="baudrate"   value="9600"/>
  <param name="loop_rate"   value="20"/>
  </node>


</launch>



